package.json

```diff
@@ -37,7 +38,7 @@
     "bcryptjs": "2.4.3",
     "body-parser": "1.18.3",
     "cheerio": "0.22.0",
-    "clone": "2.1.1",
+    "clone": "^2.1.1",
```

Comment:
why are we changing this?

Reply:
Ah didn an npm i of clone, didnt spot it in the list, will change back any remove the `^`